proxy/proxy_test.go

```diff
@@ -1482,7 +1482,7 @@ var _ = Describe("Proxy", func() {
 				Expect(string(b)).To(ContainSubstring("response_time:"))
 				Expect(string(b)).To(ContainSubstring("gorouter_time:"))
 				Expect(string(b)).To(MatchRegexp("response_time:0.1"))
-				Expect(string(b)).To(MatchRegexp("gorouter_time:0.00"))
+				Expect(string(b)).To(MatchRegexp("gorouter_time:0.0"))
```

Comment:
Shouldn't this rather be:
```suggestion
				Expect(string(b)).To(ContainSubstring("gorouter_time:0.0"))
```

Using a regex seems like a waste and the `.` will match any character.

But seeing that this is used all over the place this is probably something for a dedicated PR.

Reply:
Fair enough. I've replaced the regex calls with substring and re-ran the tests. They run just fine.